ui/src/router/activity_log/events_list/EventsList.js

```diff
@@ -19,33 +19,36 @@ export const EventsList = ({ events, status }) => {
       case "error":
         return "alert";
       default:
-        return undefined;
+        return "dot";
     }
   };
 
   return (
     <Fragment>
-      {events.map((event, idx) => (
-        <EuiComment
-          key={`event-${idx}`}
-          username={badge(event)}
-          timelineIcon={timelineIcon(event)}
-          type="update"
-          timestamp={<DateFromNow date={event.created_at} size={"s"} />}>
-          <ExpandableContainer
-            maxCollapsedHeight={43}
-            toggleKind="text"
-            isScrollable={false}>
-            <span className="activityLogEvent">{event.message}</span>
-          </ExpandableContainer>
-        </EuiComment>
-      ))}
-      {status === Status.PENDING && (
-        <EuiComment
-          username={<PulseLoader size={6} color={infoColor} />}
-          type="update"
-        />
-      )}
+      <EuiCommentList>
+        {events.map((event, idx) => (
+          <EuiComment
+            key={`event-${idx}`}
+            username={badge(event)}
+            timelineAvatarAriaLabel="System"
+            timelineAvatar={timelineIcon(event)}
+            type="update"
```

Comment:
What does `type="update"` do? I couldn't find this property in EUI documentation.

<img width="1010" alt="image" src="https://user-images.githubusercontent.com/25025366/187848097-6a7f1fcd-42d2-49e7-a4c1-491972ee2731.png">

Reply:
It's a legacy prop from an older implementation of the component. It has been removed from the latest changes that I've pushed.